### ISSUE TITLE
There are 20 marmot repositories

### ISSUE THREAD

**Issue by dsfaccini** [maxpert/marmot#1]
Including one by google
Maybe it'd be a good idea to change the name
Cool project though

**Comment by maxpert** [comment]
Interesting I didn't know Google had one too. I am open to name suggestions.  It's early days so I think I can change the name easily. The reason I named it marmot to begin with was from the idea of nodes just shouting and getting change replicated. Initially the whole thing was over UDP (I was using QUIC) but later I converged to using Raft and TCP.

**Comment by x448** [comment]
Finding an unused name is getting harder every day.  

Somewhat related because this affects module name...   In go.mod, the module name should include full prefix like this.

```
module github.com/maxpert/marmot
```

Otherwise, it won't get picked up by https://pkg.go.dev.

**Comment by maxpert** [comment]
Just did it, next version should have that as well. Changes are already on master.

**Comment by x448** [comment]
Nice.  You may need to tag a new release (not marked as pre-release).  Then you can click submit without getting the error about module name at:

https://pkg.go.dev/github.com/maxpert/marmot

It seems to ignore releases marked as pre-release.

**Comment by maxpert** [comment]
Yep there should be a build out in a while with new disk based state-machine and early version of snapshot restoration as well. I still need someone to help me test it on something other than my deployment.

**Comment by maxpert** [comment]
> It seems to ignore releases marked as pre-release.

Now I am getting 

```
"github.com/maxpert/marmot" does not have a valid module path ("marmot").
```

**Comment by x448** [comment]
If 0.3.9 was marked pre-release then maybe (just a wild guess) pkg.go.dev will continue ignoring it after it is unmarked.

You might need new release tag with new version number which wasn't used before.  Just to be sure, make a small change to the README or some other file so the new version isn't on the same commit as 0.3.9.

**Comment by maxpert** [comment]
Done I was able to get it done. I feel like some community gathering around this. May be a discord or gitter channel will help.

**Comment by disarticulate** [comment]
[marmolite](https://en.wiktionary.org/wiki/marmolite) might combine the
